### PR TITLE
Add frontend partial

### LINKED ISSUE #59: Add front end partial for square
When selecting square on the payment state page on the front end it is showing a missing template error. 

We should use the same erb code as we have used for the backend and include the javascript also in the front end.

`app/views/spree/admin/payments/source_forms/_square.html.erb`

### PR DESCRIPTION
**Pull request by garciajordy**
fix #59 

# Add Frontend Partial

As we are using the same partial for the frontend views as for the admin panel views, I copied the partial in a shared folder.

Removed our own CreatePayment service and endpoint as Solidus is already providing an endpoint to create payments on Solidus.

### CODE REVIEW

File: app/assets/javascripts/solidus_square.js [hunk 1]
```diff
@@ -12,36 +13,75 @@ document.addEventListener('DOMContentLoaded', async function () {
...
   const paymentStatusDiv = document.getElementById('payment-status-container');
   if (paymentResponse.ok) {
     document.getElementById("card-container").remove()
     document.getElementById("square-card-button").remove()
     paymentStatusDiv.innerHTML = "Payment Successfully"
+    await advanceOrder()
+
+    if (frontend) {
+      window.location.href = '/checkout';
+    }
```
[the remark this answers is not in the document]
Comment (garciajordy) [reply app/assets/javascripts/solidus_square.js]: https://github.com/nebulab/solidus_square/pull/60/commits/7a6b74a2c6cebad9c47cc7c718e34dbe1f0ca007

File: app/views/spree/api/payments/source_views/_square.json.jbuilder [hunk 2]
```diff
@@ -0,0 +1 @@
+# frozen_string_literal: true
```
[the remark this answers is not in the document]
Comment (garciajordy) [reply app/views/spree/api/payments/source_views/_square.json.jbuilder:1]: https://github.com/nebulab/solidus_square/pull/60/commits/2e9808ac272818fe2e5a15ec0bdcb7ec8af51d50

[the remark this answers is not in the document]
Comment (garciajordy) [reply app/views/spree/api/payments/source_views/_square.json.jbuilder:1]: https://github.com/nebulab/solidus_square/pull/60/commits/2e9808ac272818fe2e5a15ec0bdcb7ec8af51d50

**Review by DanielePalombo — APPROVED** [review]
Great Work!

**Pull request merged**